web/src/main/resources/META-INF/resources/webjars/common/webcert/utkast/utkastHeader/wcUtkastHeader/wcUtkastHeader.directive.js

```diff
@@ -64,8 +67,108 @@ angular.module('common').directive('wcUtkastHeader',
               }
             }
 
+            function discardModal() {
+              if(UserModel.user.origin === 'DJUPINTEGRATION' && $scope.utkastViewState.draftModel.content.name ==='dbModel') {
+                var patient = $scope.utkastViewState.intygModel.grundData.patient;
+                var dialogModel = {
+                  acceptprogressdone: false,
+                  errormessageid: 'Error',
+                  showerror: false,
+                  info: messageService.getProperty('db.label.info',
+                      {fornamn: patient.fornamn,
+                        efternamn: patient.efternamn,
+                        personnummer: patient.personId}),
+                  checkboxCheck: function() {
+                    if(document.getElementById('checkboxId').checked){
+                      //Set the disabled property to FALSE and enable the button.
+                      document.getElementById('button1id').disabled = false;
+                    } else{
+                      //Otherwise, disable the submit button.
+                      document.getElementById('button1id').disabled = true;
+                    }
+                  },
+                  checkboxId: 'checkboxId',
+                  checkboxText: 'db.label.checkbox.text'
+                };
+
+                var draftDeleteDialog = {};
+                draftDeleteDialog = dialogService.showDialog({
+                  dialogId: 'db-info-dialog',
+                  titleText: 'db.label.titleText',
+                  templateUrl: '/app/partials/dbInfo.dialog.html',
+                  bodyText: 'db.draft.label.bodyText',
+                  model: dialogModel,
+
+                  button2click: function() {
+                    $log.debug('delete draft ');
```

Comment:
Stray debug log, it should probably be removed, right?

Reply:
Fixed!